athenet/algorithm/simple_neuron_deleter2.py

```diff
@@ -0,0 +1,95 @@
+"""
+    sender - SimplE Neuron DEleteR
+
+    Deletes the least significant neurons.
+    Outgoing value of neuron is defined as sum of absolute values of
+    weights outgoing from neuron divided by sum of absolute values of
+    weights outgoing from entire layer.
+    Ingoing value of neuron is defined as sum of absolute values of
+    weights ingoing from neuron divided by sum of absolute values of
+    weights ingoing from entire layer.
+    Neuron value is defined as multiplication of its ingoing and outgoing
+    values.
+"""
+
+import numpy as np
+
+from athenet.layers import FullyConnectedLayer, ConvolutionalLayer, MaxPool
+from athenet.algorithm.utils import list_of_percentage_columns, \
+    list_of_percentage_rows, delete_column, delete_row
+
+
+def simple_neuron_deleter2(network, config):
```

Comment:
I would prefer some other change in name than 2, because it doesn't mean anything except that it was written later(at least it sounds like that)
or maybe you meant that it work on 2 values, ingoing and outgoing, in which case i would suggest something more clear